connector/exceptionsconnector/connector_metrics_test.go

```diff
@@ -174,6 +177,13 @@ func verifyConsumeMetricsInput(t testing.TB, input pmetric.Metrics, numCumulativ
 		assert.NotZero(t, dp.StartTimestamp(), "StartTimestamp should be set")
 		assert.NotZero(t, dp.Timestamp(), "Timestamp should be set")
 		verifyMetricLabels(dp, t, seenMetricIDs)
+
+		assert.Equal(t, numCumulativeConsumptions, dp.Exemplars().Len())
+		exemplar := dp.Exemplars().At(0)
+		assert.NotZero(t, exemplar.Timestamp())
+		assert.NotZero(t, exemplar.TraceID())
+		assert.NotZero(t, exemplar.SpanID())
+
```

Comment:
Would it be a good idea to add a benchmark test ?

Reply:
There's already a benchmark (`BenchmarkConnectorConsumeTraces`). Do you have anything in mind exactly to benchmark?

Reply:
I was thinking, digging Exception events in the real world workload, sort of requires deep-packet inspection and may be we add a test that essentially find 1 in 10 traces with baggage containing exception (say, as one of the 5 span event entries) and see how fast can we generate metrics and/or log events.
But I am goo with current change. I think its functionally correct and pretty awesome work. Thank You for doing this Marc.